### PR TITLE
update generated API model with Apple's updated openapi: '3.0.1', version '3.3'

### PR DESCRIPTION
**Pull request by jcmontiel**
Ran the make to update to the latest model.

### CODE REVIEW

**Comment by AvdLee** [comment]
@mathiasemil we likely want to focus on this PR instead of yours, what do you think?

**Comment by mathiasemil** [comment]
> @mathiasemil we likely want to focus on this PR instead of yours, what do you think?

I have updated #258 to use spec version 3.3 as well and re-applied the manual fixes from previous PRs. We can continue with #258 or do the same changes in this PR 🙂

**Comment by AvdLee** [comment]
Let's close this one for #258. @jcmontiel I hope you don't mind. Still highly appreciate your contribution!

File: Sources/OpenAPI/Generated/Entities/MetricsInsight.swift [hunk 1]
```diff
@@ -8,7 +8,7 @@ public struct MetricsInsight: Codable {
 	public var latestVersion: String?
 	public var metric: String?
 	public var summaryString: String?
-	public var referenceVersions: [String]?
+	public var referenceVersions: String?
```
Comment (mat1th) [inline Sources/OpenAPI/Generated/Entities/MetricsInsight.swift:11]: Be aware of the custom change in https://github.com/AvdLee/appstoreconnect-swift-sdk/pull/256/files.

File: Sources/OpenAPI/Generated/Entities/XcodeMetrics.swift [hunk 2]
```diff
@@ -126,10 +126,10 @@ public struct XcodeMetrics: Codable {
 						public var version: String?
 						public var value: Double?
 						public var errorMargin: Double?
-						public var percentageBreakdown: [PercentageBreakdownItem]?
+						public var percentageBreakdown: PercentageBreakdown?
```
Comment (mat1th) [inline Sources/OpenAPI/Generated/Entities/XcodeMetrics.swift:129]: Be aware of https://github.com/AvdLee/appstoreconnect-swift-sdk/pull/256.

**Pull request closed without merging**